File: solution_set1/2239. Find Closest Number to Zero.py
##ss
class Solution:
    def findClosestNumber(self, nums: List[int]) -> int:
        
        diff  = float('inf')
        indx = -1
        
        for x in range(len(nums)):
            if abs(nums[x] - 0) < diff:
                diff = abs(nums[x] - 0)
                indx = x
                
            elif abs(nums[x] - 0) == diff and nums[x] > nums[indx]:
                indx = x
                
        return nums[indx]
        

File: cpp_solutions/2239. Find Closest Number to Zero.cpp
class Solution {
public:
    int findClosestNumber(vector<int>& nums) {
        
        int ans = 1000000;
        
        for(int x: nums){
            if (abs(x) < abs(ans)){
                ans = x;
            }
            else if (abs(x) == abs(ans) && x > ans){
                ans = x;
            }
        }
        return ans;
        
    }
};
